### PR TITLE
[#592] Application Form: "Diversity" field

### PR DESCRIPTION
**Pull request by mkalininait**
Closes https://github.com/rails-girls-summer-of-code/rgsoc-teams/issues/592

One thing to think about: this field is not hidden during the blind phase at the moment. Together with other information people may mention their age and location there, which we kinda don't want to know before the non-blind phase. Should we ask students not to mention this info?

There was the issue "Blind selection warning" (https://github.com/rails-girls-summer-of-code/rgsoc-teams/issues/587), but it only applies to the fields which go below the 'Basics' section.

### CODE REVIEW

**Comment by mkalininait** [comment]
Screenshot:
screen shot 2017-01-16 at 00 48 07

File: app/views/application_drafts/_student_fields.html.slim [hunk 1]
```diff
@@ -5,7 +5,7 @@
...
   = s.input :application_gender_identification, label: 'Your gender self-identification', input_html: { disabled: !may_edit?(s.object) }, wrapper_html: { class: student_field_error_class(index, application_draft, :application_gender_identification) }, hint: (student_field_error_hint(index, application_draft, :application_gender_identification) ||  'No-one is excluded from applying on the basis of gender, but people who self-identify as female, or have experiences being socialized as female, are given preference during selection.')
-
+  = s.input :application_diversity, label: 'Tell us about any special circumstances which you want us to consider while reviewing your application', input_html: { disabled: !may_edit?(s.object) }, wrapper_html: { class: student_field_error_class(index, application_draft, :application_diversity) }, hint: (student_field_error_hint(index, application_draft, :application_diversity) || 'e.g. LGBTQI, people of color, people with disabilities, single parents, people taking care of family members etc.')
```
Comment (carpodaster) [inline app/views/application_drafts/_student_fields.html.slim]: Shall we make this field character-limited as well? In that case, we need a `as: :character_limited` in the form + it has be added to `Student::CHARACTER_LIMIT_FIELDS`

Comment (mkalininait) [inline app/views/application_drafts/_student_fields.html.slim]: @carpodaster I don't have a strong opinion on this. It seems to me that it's pretty hard to write more than 2000 chars about your special circumstances, but what do I know. The limit wouldn't harm anyone, I guess.

**Comment by mkalininait** [comment]
@carpodaster It would look a bit confusing, because there are the "Name", "Age" and "Location" fields in the first section, don't you think so? Maybe we make a mini-warning in the field hint?

**Comment by carpodaster** [comment]
Yeah, I guess so. Putting it as a hint seems good! :+1:

**Comment by mkalininait** [comment]
@carpodaster Done! Can I merge it now?

**Comment by carpodaster** [comment]
@mkalininait I'll merge. I will add the character limit after that.

**Pull request merged**